src/main/kotlin/calculator/Splitter.kt

```diff
@@ -0,0 +1,25 @@
+package calculator
+
+class Splitter(formula: String) {
+    val numbers: List<Float>
+
+    val operations: List<String>
+
+    init {
+        val formulas = formula.split(" ")
+        numbers = getNumbers(formulas)
+        operations = getOperations(formulas)
```

Comment:
```suggestion
        val formulas = formula.split(" ")
        val numbersAndOperations = groupByIsNumber(formulas)
        numbers = numbersAndOperations[true]!!.map { it.toFloat() }
        operations = numbersAndOperations[false]!!
```
`groupByIsNumber` 를 두번 부를 이유가 없을것 같아요~!